workflow/models.py

```diff
@@ -1745,8 +1747,9 @@ def logout_user(sender, request, user, **kwargs):
         except LoggedUser.DoesNotExist:
             pass
 
-    user_logged_in.connect(login_user)
-    user_logged_out.connect(logout_user)
+    # ! Disabling the feature...
+    # user_logged_in.connect(login_user)
```

Comment:
This also updates the `last_login` for the user on login in DB. Can we please make sure that its still being updated in DB?

Reply:
Where does it do so?

Reply:
`user_logged_in ` is a django builtin, so its being done in django. If there is custom auth, we mostly need to dispatch this signal. Just try to login on your local and check if last login gets updated in db or not.